python/python_fastapi_server/routers/edr.py

```diff
@@ -420,11 +471,16 @@ def get_params_for_collection(edr_collection: str) -> dict[str, Parameter]:
         else:
             label = param_el["name"]
 
+        symbol = Symbol(value=param_el["unit"], type=SYMBOL_TYPE_URL)
```

Comment:
From the diff, I guess there is not currently a test which shows/tests the result of this?

Reply:
Indeed. It would be good to add a test, lets see if that is feasible.

Reply:
Lets write those in https://github.com/KNMI/adaguc-server/issues/375